### ISSUE TITLE
[Bug] Robocop crashes when a comment is inbetween a multiline block

### ISSUE THREAD

**Issue by hrsc-viessmann** [MarketSquare/robotframework-robocop#821]
### What happened?

When there is a comment in the middle of a multiline block, robocop crashes instead of reporting an empty-lines-in-statement warning.

### What command/code did you try to run?

Robocop Crash Example
    Run Keywords
    ...                                        Log    Message 1   AND
    # ...                                      Log    Message 2   AND
    ...                                        Log    Message 3

### What is the full error message?

Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\Scripts\robocop.exe\__main__.py", line 7, in <module>
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\run.py", line 276, in run_robocop
    raise err
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\run.py", line 264, in run_robocop
    linter.run()
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\run.py", line 75, in run
    self.run_checks()
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\run.py", line 120, in run_checks
    found_issues = self.run_check(model, str(file))
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\run.py", line 138, in run_check
    for issue in checker.scan_file(ast_model, filename, source, templated)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\checkers\__init__.py", line 107, in scan_file
    self.visit_File(ast_model)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\checkers\__init__.py", line 112, in visit_File
    self.generic_visit(node)
  File "C:\Program Files\Python310\lib\ast.py", line 426, in generic_visit
    self.visit(item)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robot\parsing\model\visitor.py", line 49, in visit
    visitor(node)
  File "C:\Program Files\Python310\lib\ast.py", line 426, in generic_visit
    self.visit(item)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robot\parsing\model\visitor.py", line 49, in visit
    visitor(node)
  File "C:\Program Files\Python310\lib\ast.py", line 426, in generic_visit
    self.visit(item)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robot\parsing\model\visitor.py", line 49, in visit
    visitor(node)
  File "C:\Program Files\Python310\lib\ast.py", line 426, in generic_visit
    self.visit(item)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robot\parsing\model\visitor.py", line 49, in visit
    visitor(node)
  File "C:\Users\HrsC\Documents\repos\heatpump-test-automation\e3-test-automation\Setup\python_env\lib\site-packages\robocop\checkers\spacing.py", line 809, in visit_Statement
    cont = [token for token in line if token.type == "CONTINUATION"][0]
IndexError: list index out of range
Fatal exception occurred. You can create an issue at https://github.com/MarketSquare/robotframework-robocop/issues/new/choose - Thanks!

### What did you expect to happen instead?

Get an empty-lines-in-statement warning.

### Operating System

Windows

### Robocop version

3.0.0

**Comment by mnojek** [comment]
Hello @hrsc-viessmann and thanks for reporting the bug!

I found the issue and I already have a fix on the way. I will release it along with Robocop 3.1.0 as soon as possible (probably this week).

The bug reveals itself when the comment starts at the beginning of the line, like this:

```
*** Test Cases ***
Robocop Crash Example
    Run Keywords
    ...    Log Message     1   AND
# ...    Log Message     2   AND
    ...    Log Message     3
```

One more note - Robocop will not report the W1013 `empty-lines-in-statement` rule in this case, because there is no actually a blank line between statements, but a line with a comment. So no rule is violated here.

**Comment by mnojek** [comment]
FYI The bug is fixed in latest [Robocop 3.1.0 release](https://github.com/MarketSquare/robotframework-robocop/blob/master/docs/releasenotes/3.1.0.rst) :)

You can install the latest available version by:
```
pip install --upgrade robotframework-robocop
```

**Comment by hrsc-viessmann** [comment]
Thank you for the quick fix. I can confirm that the issue is fixed with 3.1.0.
